books/free-programming-books.md

```diff
@@ -1961,8 +1961,6 @@ Kerridge (PDF) (email address *requested*, not required)
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
 * [React-Bits](https://github.com/vasanthk/react-bits)
 * [React Book, your beginner guide to React](https://github.com/softchris/react-book/) - Chris Noring
-* [React Enlightenment](https://www.gitbook.com/book/frontendmasters/react-enlightenment/details)
```

Comment:
It could still be worth keeping this one, but just changing the link?

* https://www.reactenlightenment.com/
* https://github.com/FrontendMasters/react-enlightenment

Reply:
This is outdated.

Reply:
Could you elaborate on what's outdated, do you meant the resource?

Both links appear to be valid, the book itself was last updated in 2017. This seems fine to me?

Reply:
Ok, I'll add this.